src/ipa_pc/data_structures.rs

```diff
@@ -235,6 +235,41 @@ impl<G: AffineCurve> ToBytes for Proof<G> {
     }
 }
 
+/// `BatchProof` generated by batching scheme.
+/// Refers to the "Multi-poly multi-point assertions" section of the "Recursive SNARKs based on Marlin" document
```

Comment:
Let's add a reference to the paper in which this technique is presented, and not our internal one @UlrichHaboeck75